chatterbox_onnx/opm.py

```
@@ -37,6 +38,27 @@ def get_tts(self, sentence, wav_file, lang=None, voice=None):
 
+
+
+class ChatterboxTTSTransformer(TTSTransformer):
+    """ runs after TTS stage but before playback"""
+
+    def __init__(self, name="ovos-tts-transformer-chatterbox", priority=50, config=None):
+        super().__init__(name, priority, config)
+        self.engine: ChatterboxOnnx = ChatterboxOnnx()
+        self.voice = self.config["reference_voice"]
+
+    def transform(self, wav_file: str, context: dict = None) -> Tuple[str, Dict[str, Any]]:
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)